torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -374,9 +444,10 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(
     Stack& stack,
     size_t remaining_bailout_depth) {
   std::lock_guard<std::mutex> lock(compile_mutex);
-  GRAPH_DEBUG("Running ProfilingGraphExecutorImpl ", this);
+  GRAPH_DUMP("Running ProfilingGraphExecutorImpl", graph);
```

Comment:
I'd probably leave this one as GRAPH_DEBUG; since (I think) it runs on every invocation of the graph, it could dump a ton of output.  Actually, @ZolotukhinM, is this even useful?  I remember seeing a bunch of `Running ProfilingGraphExecutorImpl 0xba5eba11acc01ade` and wondering who would possibly care about the PE pointer.  Do we print it elsewhere, such that it's useful for "linking up" an execution trace?

Reply:
I think we intentionally printed the pointer here so that we could track how executors are being created during the execution (an executor in some cases could create a new executor, which could create another one, and so forth). Probably @Krovatkin could tell more about the motivation here.

For now I'd recommend leaving this place as it is and revisit it later if we want.